system_baseline/views/v1.py

```diff
@@ -661,6 +662,10 @@ def create_systems_with_baseline(baseline_id, body):
         message = str(error)
         current_app.logger.audit(message, request=request, success=False)
         raise HTTPError(HTTPStatus.BAD_REQUEST, message=message)
+    except IntegrityError:
+        message = "System(s) already associated with this baseline"
+        current_app.logger.audit(message, request=request, success=False)
```

Comment:
I believe it's better to put which system_id raised this error, since we are iterating over a `system_ids` list

Reply:
Let me give it a try

Reply:
for my understanding - will this stop all systems from getting associated and quit this try block even if one system is already associated?